pennylane_lightning/core/src/simulators/lightning_gpu/measurements/MeasurementsGPU.hpp

```diff
@@ -273,7 +273,7 @@ class Measurements final
         PL_CUSTATEVEC_IS_SUCCESS(custatevecSamplerSample(
             this->_statevector.getCusvHandle(), sampler, bitStrings.data(),
             bitOrdering.data(), bitStringLen, rand_nums.data(), num_samples,
-            CUSTATEVEC_SAMPLER_OUTPUT_ASCENDING_ORDER));
+            CUSTATEVEC_SAMPLER_OUTPUT_RANDNUM_ORDER));
```

Comment:
What is the implication of this change and why do we need that? I'm curious.

Reply:
This homogenizes the behaviour of LGPU with LQ and LK. This is useful to prevent possible bugs when `shots` is a List and the device sets has_partitioned_shots = True

Reply:
[Original post](https://github.com/PennyLaneAI/pennylane-lightning/pull/892/files#r1754783539)

Reply:
Thank you